crates/bevy_remote/src/http.rs

```diff
@@ -36,6 +37,37 @@ pub const DEFAULT_PORT: u16 = 15702;
 /// The default host address that Bevy will use for its server.
 pub const DEFAULT_ADDR: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
 
+/// A struct that holds a collection of HTTP headers.
+///
+/// This struct is used to store a set of HTTP headers as key-value pairs, where the keys are
+/// of type [`HeaderName`] and the values are of type [`HeaderValue`].
+///
+#[derive(Debug, Resource, Clone)]
+pub struct Headers {
```

Comment:
If you used a type alias instead you could get all the `get`, `insert` and `remove` methods for free but again not a blocker.